src/components/TextField/__stories__/TextField.stories.mdx

```diff
@@ -167,30 +205,52 @@ Users can insert text.
   </Story>
 </Canvas>
 
+### Input field with description icon
 
-### Input field with date
 <Canvas>
-  <Story name="Input field with date">
+  <Story name="Input field with description">
+    <div className="monday-storybook-text-field_size">
+      <TextField title="I need help" description="Enter Text" placeholder="Text" size={TextField.sizes.MEDIUM} />
+    </div>
+  </Story>
+</Canvas>
+
+### Input field with description + icon
+
+<Canvas>
```

Comment:
let's remove this from the docs because changing the icon is not a best practice from Vibe perspective and we do not want to encourage teams to use it